warehouse/forklift/legacy.py

```diff
@@ -571,6 +604,12 @@ class MetadataForm(forms.Form):
         description="Requires-Dist",
         validators=[wtforms.validators.Optional(), _validate_legacy_dist_req_list],
     )
+    provides_extra = (
```

Comment:
From https://github.com/pypi/warehouse/pull/11380#discussion_r1033923344

This needs to be added as a field of the Release model as well: https://github.com/pypi/warehouse/blob/main/warehouse/packaging/models.py#L423-L604

Reply:
I think this can be a generic `sqlalchemy.dialects.postgresql.ARRAY` of `sqlalchemy.Text` since the incoming are validated:

```
    provides_extra = Column(ARRAY(Text))
```